aiida/cmdline/commands/cmd_storage.py

```diff
@@ -84,42 +84,100 @@ def storage_integrity():
 @click.option('--statistics', is_flag=True, help='Provides more in-detail statistically relevant data.')
 def storage_info(statistics):
```

Comment:
This now needs to use/adapt `from aiida.cmdline.utils.common import get_database_summary`